docs/index.rst

```diff
@@ -50,10 +50,10 @@ Description
 
 `buildtest <https://github.com/buildtesters/buildtest>`_  is
 a testing framework to help HPC sites write test for their system as part of their
-routine acceptance & regression test. buildtest provides a YAML interface to write tests
-which buildtest can validate and generate shell scripts that can run on your HPC system.
-The test template (YAML) is called **buildspec** which can define one or more test instance
-that is validated by a `json schema <https://json-schema.org/>`_. buildtest supports the following batch schedulers:
+routine acceptance and regression testing. Buildtest provides a YAML interface to write tests
+which buildtest can validate and then use to generate shell scripts that run on your HPC system.
+The test template (YAML) is called a **buildspec** which can define one or more test instances
```

Comment:
Perhaps we can rewrite this sentence. What you think?

*In buildtest, tests are written declaratively in YAML known as **buildspec**, which can define one or more test instances and each test instance is validated by a [json schema](https://json-schema.org/).*

Reply:
This sentence like this is awkward because the subject is too far removed and the structure of the second half is ambiguous.  If you didn't know buildtest, buildspec and YAML already, it would be very difficult to decode this sentence. Better to split it up, keep it simple and make it clear.